content/post/37-ipfs-conf-dev-meetings-and-user-registry.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2018-05-19
+url: 37-ipfs-conf-dev-meetings-and-user-registry.md
+title: Announcements: IPFS Conf,  IPFS & libp2p Developer Meetings and a User Registry
```

Comment:
This makes yml belive that you're trying to create a map, wrap the title in quotes and it'll work! 

```
title: "Announcements: IPFS Conf, IPFS & libp2p Developer Meetings and a User Registry" 
```